packages/frontend/src/context/Post.ts

```diff
@@ -594,6 +598,11 @@ export class Data {
                         this.loadCommentsByPostId(_comment.postId),
                     ])
                     this.save()
+
+                    return {
+                        id: _comment.postId + '#' + transaction,
```

Comment:
The url should be postID (nanoID) # commentID (nanoID), is it?
so it should be `id: _comment.postId + '#' + _comment._id `